apps/betterangels-backend/shelters/permissions.py

```diff
@@ -0,0 +1,118 @@
+
+
+class EntryRequirementPermissions(models.TextChoices):
+    ADD = "shelters.add_entryrequirement", _("Can add entry requirement")
+    CHANGE = "shelters.change_entryrequirement", _("Can change entry requirement")
+    DELETE = "shelters.delete_entryrequirement", _("Can delete entry requirement")
+    VIEW = "shelters.view_entryrequirement", _("Can view entry requirement")
+
+
+class FunderPermissions(models.TextChoices):
+    ADD = "shelters.add_funder", _("Can add funder")
+    CHANGE = "shelters.change_funder", _("Can change funder")
+    DELETE = "shelters.delete_funder", _("Can delete funder")
+    VIEW = "shelters.view_funder", _("Can view funder")
+
+
+class GeneralServicePermissions(models.TextChoices):
+    ADD = "shelters.add_generalservice", _("Can add general service")
+    CHANGE = "shelters.change_generalservice", _("Can change general service")
+    DELETE = "shelters.delete_generalservice", _("Can delete general service")
+    VIEW = "shelters.view_generalservice", _("Can view general service")
+
+
+class HealthServicePermissions(models.TextChoices):
+    ADD = "shelters.add_healthservice", _("Can add health service")
+    CHANGE = "shelters.change_healthservice", _("Can change health service")
+    DELETE = "shelters.delete_healthservice", _("Can delete health service")
+    VIEW = "shelters.view_healthservice", _("Can view health service")
+
+
+class ImmediateNeedPermissions(models.TextChoices):
+    ADD = "shelters.add_immediateneed", _("Can add immediate need")
+    CHANGE = "shelters.change_immediateneed", _("Can change immediate need")
+    DELETE = "shelters.delete_immediateneed", _("Can delete immediate need")
+    VIEW = "shelters.view_immediateneed", _("Can view immediate need")
+
+
+class ParkingPermissions(models.TextChoices):
+    ADD = "shelters.add_parking", _("Can add parking")
+    CHANGE = "shelters.change_parking", _("Can change parking")
+    DELETE = "shelters.delete_parking", _("Can delete parking")
+    VIEW = "shelters.view_parking", _("Can view parking")
+
+
+class PetPermissions(models.TextChoices):
+    ADD = "shelters.add_pet", _("Can add pet")
+    CHANGE = "shelters.change_pet", _("Can change pet")
+    DELETE = "shelters.delete_pet", _("Can delete pet")
+    VIEW = "shelters.view_pet", _("Can view pet")
+
+
+class PopulationPermissions(models.TextChoices):
+    ADD = "shelters.add_population", _("Can add population")
+    CHANGE = "shelters.change_population", _("Can change population")
+    DELETE = "shelters.delete_population", _("Can delete population")
+    VIEW = "shelters.view_population", _("Can view population")
+
+
+class ShelterPermissions(models.TextChoices):
+    ADD = "shelters.add_shelter", _("Can add shelter")
+    CHANGE = "shelters.change_shelter", _("Can change shelter")
+    DELETE = "shelters.delete_shelter", _("Can delete shelter")
+    VIEW = "shelters.view_shelter", _("Can view shelter")
+
+
+class ShelterFieldPermissions(models.TextChoices):
```

Comment:
Was debating on whether to combine this into shelter perms or create a new one.  Opted for field permissions since that groups all field perms into a single enum.